keyboards/handwired/macroboard/keymaps/via/config.h

```diff
@@ -0,0 +1 @@
+#define DYNAMIC_KEYMAP_LAYER_COUNT 6
```

Comment:
this needs a license header and an include guard.

Reply:
done